Add TSC call details to the SPIFFE README

evan2645 opened the pull request:
Now that the TSC calls are public, they should be documented on the SPIFFE readme. This PR does that.

Signed-off-by: Evan Gilman <egilman@vmware.com>

evan2645 committed:
Add TSC call details to the SPIFFE README

Signed-off-by: Evan Gilman <egilman@vmware.com>

spikecurtis commented on README.md:
```diff
@@ -55,7 +55,7 @@ SPIFFE is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
 * [GOVERNANCE](/GOVERNANCE.md)
 
-### SIGs & Working Groups<a name="sigs"></a>
+### SIGs & Working Groups
```
I'm guessing the anchor was in place to allow urls with a fragment `#sigs`. Since those URLs might still be in the wild, I don't see a reason to break them.

evan2645 replied:
Fixed

evan2645 committed:
Add anchor tag back to README section
This anchor tag looked out of place as it's the only one in the whole
README, and it's not markdown. Git blame shows that it was introduced in
the commit immediately following the initial commit, and that at that
time, all headings had a similar anchor tag. Believing it to be
spurious, I removed it in my previous commit, however it was pointed out
during review that there may be URLs out there using this tag as a
fragment, and that we could be breaking them by removing it... so, this
commit restores the tag.

Maybe someone in the future can do the homework to remove it :)

Signed-off-by: Evan Gilman <egilman@vmware.com>

evan2645 committed:
Merge branch 'master' into add-tsc-call-details-to-readme
